generator/exceptions.py

```diff
@@ -21,5 +21,17 @@
 # SOFTWARE.
 
 
-class GeneratorDeployException(Exception):
+class SandboxException(Exception):
+    """ There was an issue during execution. """
+
+
+class GeneratorDeployException(SandboxException):
     pass
+
+
+class SandboxCommandFailed(SandboxException):
+    """ The command executed in sandbox failed. """
+
+    def __init__(self, output: str, reason: str):
+        self.output: str = output
+        self.reason: str = reason
```

Comment:
what about to add there also RC?

Reply:
I'd say that it's irrelevant to some extent: it's enough to know that the command failed and we have the output, but it's definitely a good suggestion - I'll add it.

The other problem is that it's buried in the super-complex structure, so it won't be trivial to dig it. Let's give it a shot and we shall see.